src/parquet/column/writer.cc

```diff
@@ -99,13 +101,15 @@ void ColumnWriter::WriteNewPage() {
   // TODO(PARQUET-590): Encodings are hard-coded
   int64_t bytes_written = pager_->WriteDataPage(num_buffered_values_,
       num_buffered_encoded_values_, definition_levels, Encoding::RLE, repetition_levels,
-      Encoding::RLE, values, Encoding::PLAIN);
+      Encoding::RLE, values, Encoding::PLAIN, page_statistics_);
   total_bytes_written_ += bytes_written;
 
   // Re-initialize the sinks as GetBuffer made them invalid.
   InitSinks();
   num_buffered_values_ = 0;
   num_buffered_encoded_values_ = 0;
+  chunk_statistics_->Merge(*page_statistics_);
+  page_statistics_->Reset();
```

Comment:
Looking further down the review it seems that this will always be non-null, so maybe this is moot.